pkg/apply/merge.go

```diff
@@ -112,6 +116,51 @@ func MergeServiceAccountForUpdate(current, updated *unstructured.Unstructured) e
 	return nil
 }
 
+// MergeWebhookConfiguration ensure caBundle is keept at webhooks's clientConfig
+func MergeWebhookConfiguration(current, updated *unstructured.Unstructured) error {
```

Comment:
does not need to be public. You can test it through the public `MergeObjectForUpdate`